### PR TITLE
remove V1 network protocol and unused code

### LINKED ISSUE #813: Remove v1 protocol

### PR DESCRIPTION
**Pull request by woutslakhorst**
closes #813

### CODE REVIEW

**Review by gerardsn — COMMENTED** [review]
Looks good to me, just some questions

File: network/cmd/cmd.go [hunk 1]
```diff
@@ -53,10 +53,6 @@ func FlagSet() *pflag.FlagSet {
...
 	flagSet.String("network.nodedid", defs.NodeDID, "Specifies the DID of the organization that operates this node, typically a vendor for EPD software. It is used to identify the node on the network. If the DID document does not exist of is deactivated, the node will not start.")
 	flagSet.IntSlice("network.protocols", defs.Protocols, "Specifies the list of network protocols to enable on the server. They are specified by version (1, 2). If not set, all protocols are enabled.")
-	flagSet.Int("network.v1.adverthashesinterval", defs.ProtocolV1.AdvertHashesInterval, "Interval (in milliseconds) that specifies how often the node should broadcast its last hashes to other nodes.")
-	flagSet.Int("network.v1.advertdiagnosticsinterval", defs.ProtocolV1.AdvertDiagnosticsInterval, "Interval (in milliseconds) that specifies how often the node should broadcast its diagnostic information to other nodes (specify 0 to disable).")
-	flagSet.Int("network.v1.collectmissingpayloadsinterval", defs.ProtocolV1.CollectMissingPayloadsInterval, "Interval (in milliseconds) that specifies how often the node should check for missing payloads and broadcast its peers for it (specify 0 to disable). "+
```
[the remark this answers is not in the document]
Comment (woutslakhorst) [reply network/cmd/cmd.go:58]: not at the config level, it uses an incremental backoff. Jobs are stored in payload_jobs.db

File: README.rst [hunk 2]
```diff
@@ -166,55 +173,52 @@ The following options can be configured on the server:
...
+events.nats.timeout                30                                                                                                                                                                                                                                                                                                               Timeout for NATS server operations
+**JSONLD**
+jsonld.contexts.localmapping       [https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v1.json=assets/contexts/lds-jws2020-v1.ldjson,https://schema.org=assets/contexts/schema-org-v13.ldjson,https://nuts.nl/credentials/v1=assets/contexts/nuts.ldjson,https://www.w3.org/2018/credentials/v1=assets/contexts/w3c-credentials-v1.ldjson]  This setting allows mapping external URLs to local files for e.g. preventing external dependencies. These mappings have precedence over those in remoteallowlist.
```
[the remark this answers is not in the document]
Comment (woutslakhorst) [reply README.rst:204]: does not matter,  I think the order is random...

**Review by gerardsn — APPROVED** [review]

**Comment by woutslakhorst** [comment]
@reinkrul comments? Since it's a lot of your code being removed. Maybe you have some pointers on dead-code?

File: README.rst [hunk 3]
```diff
@@ -48,13 +48,20 @@ After the services have started you can try the following endpoints:
 Development
 ^^^^^^^^^^^
 
-Go >= 1.18 is required.
+Go >= 1.17 is required.
```
Comment (reinkrul) [inline README.rst:51]: `go.mod` states 1.18

**Comment by reinkrul** [comment]
`network_integration_test` still mentions v1 and has specific v2 tests, is it still useful to have non-v2 tests and v2 tests since we only have v2 now?

**Review by reinkrul — CHANGES_REQUESTED** [review]

**Review by reinkrul — APPROVED** [review]

**Pull request merged**